tests/test_context.py

```diff
@@ -11,6 +11,12 @@ def dummy(t=0):
     return True
 
 
+@asyncio.coroutine
+def adds_to_context():
```

Comment:
The scope of this function is the `test_propagates_copy_of_context`, IMHO I would prefer to move it as a closure of that function.